apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/ListJobsOptions.cs

```diff
@@ -56,7 +61,7 @@ internal void ModifyRequest(ListRequest request)
                 request.AllUsers = AllUsers;
             }
 
-            // TODO: Projection?
+            request.Projection = Projection;
```

Comment:
Please change this to only set it if the value is non-null. While we don't compose options anywhere at the moment, I can imagine doing so at some point.

Reply:
Happy to do so for clarity, but request.Projection is nullable, that's why I didn't check before.

Reply:
Done, I think I get what you are saying now, in case in the future we were to surface request.Projection as a combination of different properties in our options class, right? Or is there something I'm still missing?

Reply:
It's more if we get into a situation where we want to merge two options objects - having null mean "don't change anything" makes that merge easy. That's what we use in CallSettings for gRPC-based projects a lot.

Reply:
OK, I get it now. Thanks!